e2e/cypress/integration/messaging/message_parse_spec.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// [number] indicates a test step (e.g. # Go to a page)
+// [*] indicates an assertion (e.g. * Check the title)
+// Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Login as "user-1" and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M17444 - correctly parses "://///" as Markdown and does not break the channel', () => {
+        // # Go to Town Square as test channel
+        cy.get('#sidebarItem_town-square').click({force: true});
+
+        // # Validate if the channel has been opened
+        cy.url().should('include', '/channels/town-square');
+
+        // # type in the message "://///"
+        const message = '://///';
+        const textAfterParsed = message.substr(2);
+        cy.postMessage(message);
+
+        // # check if message sent correctly, it should parse it as 😕////"
+        cy.getLastPostId().then((postId) => {
+            verifyPostedMessage(postId, textAfterParsed);
```

Comment:
does this fail if `textAfterParsed` doesn't match `postId`?

Reply:
hey @ali-farooq0 - not completely sure what you mean. `postId` is used to get the element and `textAfterParsed` is used to match

Reply:
what happens when the text doesn't match? does the test fail here?